example_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials_test
+
+import (
+	"context"
+
+	credentials "github.com/oras-project/oras-credentials-go"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func ExampleNewNativeStore() {
+	// For the argument of NewNativeStore, use "wincred" for Windows,
```

Comment:
We may need to reword this comment. The argument can be the native keychain listed above and any program that follows the `docker-credentials-helper` protocol.
Reference: https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Reply:
changed this comment.